pkg/resources/database.go

```diff
@@ -0,0 +1,145 @@
+package resources
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/pkg/errors"
+)
+
+func Database() *schema.Resource {
+	d := newResourceDatabase()
+	return &schema.Resource{
+		Create: d.Create,
+		Read:   d.Read,
+		Delete: d.Delete,
+		Update: d.Update,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    false,
+				Description: "TODO",
+				// TODO validation
+			},
+			"comment": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "",
+				// TODO validation
+			},
+		},
+	}
+}
+
+type database struct{}
+
+func newResourceDatabase() *database {
+	return &database{}
+}
+
+func (d *database) Create(data *schema.ResourceData, meta interface{}) error {
+	name := data.Get("name").(string)
+	comment := data.Get("comment").(string)
+	db := meta.(*sql.DB)
+
+	// TODO escape name
+	// TODO escape comment
+	// TODO name appears to get normalized to uppercase, should we do that? or maybe just consider it
+	// 	case-insensitive?
+	stmt := fmt.Sprintf("CREATE DATABASE %s COMMENT='%s'", name, comment)
```

Comment:
would this allow people to do terrible things?

Reply:
Without escaping, yes.

Reply:
I will follow up with a PR just on escaping. Need to research more.